src/ownclouddolphinpluginhelper.cpp

```diff
@@ -119,9 +120,11 @@ void OwncloudDolphinPluginHelper::slotReadyRead()
             _paths.append(file);
             continue;
         } else if (command == QByteArrayLiteral("STRING")) {
-            auto args = QString::fromUtf8(info).split(':');
+            auto args = info.split(':');
```

Comment:
This change looks fishi

Reply:
No, it is not. `info` is  a bytearray, which has a split function. There is no need to convert it to QString imho.

Reply:
But you remove the one to String conversation and replaced it by multiple in the next line

Reply:
Also https://github.com/owncloud/client-desktop-shell-integration-dolphin/pull/1/files?diff=unified&w=0#diff-34af974cf3f2851c0dd15423078bfec64abdcb1a72abd1e58d6eee8a9b0e559bR75

Reply:
i thought it is easier to read, because you do not need to know that mid(1) returns everything to the end.

The `strings[args[0]] = ...` is not longer possible in Qt6.

I do not get the _Also..._ concern you have, please be more verbose.

Reply:
You changed the map to use QByteArray as key type. Thats why `strings[args[0]] =` doesn't work anymore.

Reply:
And the old code was the idiomatic qt solution.

Reply:
See https://github.com/owncloud/client-desktop-shell-integration-dolphin/pull/1/files#diff-fb47628cb5c913543faa69e12348874669bafa1b20b25e1dc7f1ce3a6e5e318cR77

Reply:
new variant, maybe you like that. 

The QByteArray as key type saves other QStringLiteral() at other places which I find annoying.